service/src/main/java/greencity/service/notification/NotificationServiceImpl.java

```diff
@@ -51,25 +53,50 @@
 @AllArgsConstructor
 @Slf4j
 public class NotificationServiceImpl implements NotificationService {
-    private UserRepository userRepository;
-    private UserNotificationRepository userNotificationRepository;
-    private BagRepository bagRepository;
-    private OrderRepository orderRepository;
-    private ViolationRepository violationRepository;
-    private NotificationParameterRepository notificationParameterRepository;
+    private final UserRepository userRepository;
+    private final UserNotificationRepository userNotificationRepository;
+    private final BagRepository bagRepository;
+    private final OrderRepository orderRepository;
+    private final ViolationRepository violationRepository;
+    private final NotificationParameterRepository notificationParameterRepository;
     @Autowired
     @Qualifier("kyivZonedClock")
     private Clock clock;
-    private List<? extends AbstractNotificationProvider> notificationProviders;
+    private final List<? extends AbstractNotificationProvider> notificationProviders;
     private final NotificationTemplateRepository templateRepository;
-    @Autowired
-    private final InetAddressProvider inetAddressProvider;
-
     @Autowired
     @Qualifier("singleThreadedExecutor")
     private ExecutorService executor;
 
+    private final Environment environment;
```

Comment:
Instead of using the Environment class Please add your properties here: https://github.com/ita-social-projects/GreenCityUBS/blob/dev/core/src/main/java/greencity/configuration/RedirectionConfigProp.java
Or create a new one if needed by following the example.